src/main/java/build/buf/protovalidate/internal/celext/CustomOverload.java

```diff
@@ -72,20 +82,20 @@ static Overload[] create() {
    *
    * @return The {@link Overload} instance for the "format" operation.
    */
-  private static Overload binaryFormat() {
+  private static Overload format() {
     return Overload.binary(
-        "format",
+        OVERLOAD_FORMAT,
         (lhs, rhs) -> {
-          if (rhs.type() != ListT.ListType) {
-            return Err.newErr("format: expected list");
+          if (lhs.type().typeEnum() != TypeEnum.String || rhs.type().typeEnum() != TypeEnum.List) {
+            return Err.noSuchOverload(lhs, OVERLOAD_FORMAT, rhs);
           }
           ListT list = (ListT) rhs.convertToType(ListT.ListType);
-          String formatString = lhs.value().toString();
-          Val status = Format.format(formatString, list);
-          if (status.type() == Err.ErrType) {
-            return status;
+          String formatString = (String) lhs.value();
+          try {
+            return StringT.stringOf(Format.format(formatString, list));
+          } catch (Err.ErrException e) {
+            return e.getErr();
```

Comment:
Updated this to use exceptions for control flow instead of dealing with interpreting the returned Val type (either Err or String).